system_strings/sysmes_text.en

```diff
@@ -538,28 +538,28 @@ I head downstairs.
 In the Depths of Hell
 Reason
 To Vlov's Stronghold
-天国の梯子
-炎の手
-ノエルを助ける。
-ありえない切断
-ノエルを見捨てる。
-――――このまま押し切る！
-限界だ、一度呼吸を……！
-二十七祖
-先輩に駆け寄る。
-先生に駆け寄る。
-ノエル先生の個人授業
-地底からの脱出
-アルクェイドさんご立腹
-霧の中
-苛立ち
-冷気の檻
-シエル先輩を見る。
-ヴローヴを見る。
-凍土のワルツ
-安息こそ故郷
-おやすみなさい。
-後処理
+The ladder to paradise
+Hands of flame
+Save Noel.
+Impossible severance
```

Comment:
This is Shiki slicing Vlov's arm off. Amputation or something might be more apt.